examples/federated-learning/tff/nvflare/docker-image/Dockerfile

```diff
@@ -0,0 +1,8 @@
+#checkov:skip=CKV_DOCKER_3: No need for a user
```

Comment:
Explain why we don't need a user. Seems a valid warning.

Reply:
Done